live-examples/js-examples/date/date-toprimitive.html

```diff
@@ -0,0 +1,13 @@
+<pre>
+<code id="static-js" class="language-js">
+  const date = new Date('20 December 2019 14:48'); //initialize a new Date object
+
+  console.log(date[Symbol.toPrimitive]('string')); //log primitive of date with hint as string
+  //expected output: "Fri Dec 20 2019 20:18:00 GMT+0530 (India Standard Time)"
+  //note: the timezone specific data(GMT+0530 (India Standard Time)) may vary according to your location
```

Comment:
Actually the whole thing may vary, so it would be better to say:

`// timezone specific data may vary according to your local settings`

Reply:
I don't think the hours and minutes will vary because, i haven't specified timezone in the date declaration `'20 December 2019 14:48 U̶T̶C̶'` so it will be inferred by the runtime according to the locale specifications, and what do you mean by local settings ?  Thanks.

Reply:
I'm in Pacific time, and I do see hours and minutes that differ from `expected output`:

<img width="712" alt="Screen Shot 2020-01-07 at 8 25 35 AM" src="https://user-images.githubusercontent.com/432915/71910643-27d55780-3127-11ea-8267-e3b3857fe8dc.png">

So perhaps the expected output is incorrect?

Reply:
yep the expected output is incorrect I'm getting the following result.
blob:null/472b3468-0753-fe44-bcad-544c9ba8f324

Reply:
fixed it.